### DISCUSSION TITLE
Disable on mobile posibility (or allow scroll on mobile)

### DISCUSSION THREAD

**Discussion by laurensV** [nolimits4web/atropos#10] [Ideas]
For big boxes that cover almost your entire screen on mobile, it is impossible to get past it, as you cannot scroll anymore on the atropos element on mobile. Would be great if either scroll on mobile would be fixed, or an option to disable on mobile

EDIT: nvm, you can already do this with rotateTouch function, great!

**Comment by nolimits4web** [comment]
That is why there is a `rotateTouch` parameter where you can disable it or set to `scroll-y` value

**Reply by strsar** [reply]
You know what's pretty awesome?

The first 2 questions I pondered to myself trying this out are the only 2 questions asked and answered here... Like a boss... Way to be proactive here and appreciate the work! Will be attempting to pepper this into a production project in the near future. 

Seems to play pretty well with swiper too. 

Just saying thanks and this topic could probably be marked as "Answered"!

_(not to get too critical here w/ minor feedback, but i did notice after applying `rotateTouch:  'scroll-y'` (testing on ios) that it takes one extra interaction to break out of the listeners after you've started interaction horizontally. the next touch move over the element still triggers atropos regardless of if it's horizontal or vertical)_
